go-controller/pkg/ovn/policy_common.go

```diff
@@ -30,6 +30,19 @@ type namespacePolicy struct {
 	deleted         bool //deleted policy
 }
 
+func NewNamespacePolicy(policy *knet.NetworkPolicy) *namespacePolicy {
```

Comment:
not really related to the rest of the PR...

Reply:
Ok.